content/kubermatic/master/tutorials-howtos/metering/_index.en.md

```diff
@@ -95,17 +99,35 @@ kind: Seed
 spec:
   metering:
     enabled: true
-    # StorageClassName is the name of the storage class that the metering tool uses to save processed files before
-    # exporting it to s3 bucket. Default value is kubermatic-fast.
-    storageClassName: kubermatic-fast
+    # StorageClassName is the name of the storage class that the metering prometheus instance uses to store metric data for reporting.
+    storageClassName: ""
     # StorageSize is the size of the storage class. Default value is 100Gi.
     storageSize: 100Gi
+    reports:
+      weekly:
+        interval: 7
+        schedule: 0 1 * * 6
```

Comment:
We could add retention to the example:
```suggestion
        interval: 7
        retention: 90
        schedule: 0 1 * * 6
```

Reply:
I added the section about report configuration via the dashboard. IMO this should be ok. Types are kind of self explaining.